controller/volunteer_controller.py

```diff
@@ -70,6 +72,8 @@ def edit_phone(volunteer: Volunteer, phone: str) -> Volunteer:
 
 def edit_camp(volunteer: Volunteer, camp: Camp, is_admin: bool) -> Volunteer:
     old_camp = volunteer.camp
+    if camp.plan.is_closed:
+        raise ControllerError(f"Plan {camp.plan.name} is closed. Please choose another plan.")
```

Comment:
suggest skipping 'Please choose another plan.', cuz it may make more sense to let the interface handle that depending on the context

Reply:
Done